src/app/services/register/register.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RegisterService {
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  sendRegistrationInfo (username: string, password: string, confirmPsw: string): Observable<any> {
```

Comment:
Same issue here as with my login service, the service shouldn't provide the complete response, only the necessary portions.

Reply:
solved thanks to you